awscodecommit/Dockerfile

```diff
@@ -1,25 +1,31 @@
-FROM golang:alpine AS builder
+FROM golang:1.13-alpine AS builder
+
+ENV GO111MODULE on
+ENV CGO_ENABLED 0
+ENV GOOS linux
+ENV GOARCH amd64
```

Comment:
(nit) I know it's less pretty, but we could minimize the number of metadata layers with
```dockerfile
ENV \
    GO111MODULE=on \
    CGO_ENABLED=0 \
    GOOS=linux \
    GOARCH=amd64
```
I don't feel strongly about it though.

Reply:
`ENV` instructions no longer result in filesystem layers. Since Docker 1.10 only the instructions `RUN`, `COPY`, `ADD` create layers. In the interest of making the `Dockerfile` more readable i split those `ENV` and `LABEL` instructions

source: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers

Reply:
I missed that update, thanks for the pointers!